bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat

```diff
@@ -0,0 +1,272 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  500000	= nevents ! Number of unweighted events requested 
```

Comment:
for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000